Makefile

```diff
@@ -109,11 +109,13 @@ clean:
 	rm -f man/avocado.1
 	rm -f man/avocado-rest-client.1
 	rm -rf docs/build
+	rm -rf avocado.egg-info
```

Comment:
This is the `clean` target, but now I noticed that I'm removing it twice... Fixing this.